Userland/Libraries/LibJS/Runtime/AbstractOperations.cpp

```diff
@@ -156,4 +159,30 @@ Object* get_prototype_from_constructor(GlobalObject& global_object, Function con
     return &prototype.as_object();
 }
 
+// 19.2.1.1 PerformEval ( x, callerRealm, strictCaller, direct )
+Value perform_eval(VM& vm, Value x, GlobalObject& caller_realm, CallerMode strict_caller, EvalMode direct)
+{
+    assert(direct == EvalMode::Direct || strict_caller == CallerMode::NonStrict);
```

Comment:
Serenity uses `VERIFY` instead of `assert`
```suggestion
    VERIFY(direct == EvalMode::Direct || strict_caller == CallerMode::NonStrict);
```